flink-staging/flink-streaming/flink-streaming-scala/src/test/java/org/apache/flink/streaming/scala/api/CsvOutputFormatITCase.java

```diff
@@ -36,6 +37,10 @@ protected void preSubmit() throws Exception {
 	@Override
 	protected void testProgram() throws Exception {
 		OutputFormatTestPrograms.wordCountToCsv(WordCountData.TEXT, resultPath);
+		OutputFormatTestPrograms.wordCountToCsv(WordCountData.TEXT, resultPath, 1);
```

Comment:
Is this actually working? 
I thought by default files are not overwritten. So writing to the same file again, should not work and the second job should fail.